### PR TITLE
Imagebuilder - ensure AMI disks are encrypted with KMS

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/checks/resource/aws/ImagebuilderImageRecipeEBSEncrypted.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        name = "Ensure that Image Recipe EBS Disk are encrypted with CMK"
+        id = "CKV_AWS_200"
+        supported_resources = ["aws_imagebuilder_image_recipe"]
+        categories = [CheckCategories.ENCRYPTION]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
+        if conf.get('block_device_mapping'):
+            mappings = conf.get('block_device_mapping')
+            for mapping in mappings:
```
Comment (gruebel) [inline checkov/terraform/checks/resource/aws/ImagebuilderImageRecipeEBSEncrypted.py:19]: I think you current code doesn't consider that the first block device is encrypted, but the second or any follow is not. Cann you add this test case and check, if you need to adjust your code.

**Review by gruebel — COMMENTED** [review]
probably you also need to merge master into your branch to make the ID check pass. And congrats for creating the 200th check 🎉 🍾

File: checkov/terraform/checks/resource/aws/ImagebuilderImageRecipeEBSEncrypted.py (hunk 1 above)
Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/aws/ImagebuilderImageRecipeEBSEncrypted.py:19]: good catch. have it now

**Review by gruebel — APPROVED** [review]
🍻

**Pull request merged**